.github/workflows/wasm.yaml

```diff
@@ -45,6 +45,13 @@ jobs:
       - name: Setup emsdk
         uses: mymindstorm/setup-emsdk@v7
 
+      - name: Cache geckodriver
+        id: cache-lel
+        uses: actions/cache@v3
+        with:
+          path: /opt/hostedtoolcache/geckodriver/
+          key: wasm_geckodriver
+
       - name: Set up gecko driver
```

Comment:
If the cache was a hit, do we need to skip the setup step to avoid overwriting the cache and hitting the API key again?